app/routes/print.js

```diff
@@ -29,8 +29,8 @@ export async function loader({ request }) {
         },
       }
     );
-  const orderDetails = await response.json();
-  const order = orderDetails.data.order;
+  const orderData = await response.json();
```

Comment:
nitpick but could it be simplified if we do: 
```ts
const response = await admin.graphql(
      ...
    ).then( data => data.json() );

const { order } = response.data;
```

Reply:
Yeah, but I feel like people prefer `async`/`await` over `.then` 🤷 . The variable name change is pretty minor as well. It's just something that came up during the walkthrough.